### ISSUE TITLE
Question about FLD value

### ISSUE THREAD

**Issue by Danee-wawawa** [marcojira/fls#7]
The FLD I calculated is negative(for example “-1581.340”). Is this wrong? I see that the range you wrote is [0, 100].

**Comment by marcojira** [comment]
Yes a negative value indicates that your generated samples give a MoG with better likelihood than the train samples. Could possibly be due to providing insufficient training samples? Could you detail the train, test and generated sets of features you used?

**Comment by Danee-wawawa** [comment]
Thank you for the reply. I use cyclegan model to transfer the style of A to B. A is the content images, and B is the style images. A and B are unpaired data. When conducting FLD evaluation, the train set and test set are divided from the B data, and the generated data set is the data after the style transfer of the A data. And the training set data size is 2k, the test set data size is 1k, and the generated data size is 3k. Not sure if structuring the data this way is correct?

**Comment by marcojira** [comment]
Could you try with a generated data size of 1k? What's happening now is that to get a baseline LL, FLD is splitting the train set in two and using half as MoG (i.e. in your case 1k samples).

**Comment by Danee-wawawa** [comment]
OK, I will try it, thank you~
